scripts/restart_mcp_server.sh

```diff
@@ -6,19 +6,67 @@ CONTAINER_NAME=${CONTAINER_NAME:-stt-mcp-server-linux}
 TMUX_SESSION=${TMUX_SESSION:-claude}
 TMUX_TMPDIR=${TMUX_TMPDIR:-"$HOME"/.tmux}
 
-docker stop "$CONTAINER_NAME" || true
-docker rm "$CONTAINER_NAME" || true
+# Check if Docker is running
+if ! docker info >/dev/null 2>&1; then
+    echo "Error: Docker is not running or not accessible" >&2
+    exit 1
+fi
+
+# Stop and remove container if it exists
+if docker ps -a --format '{{.Names}}' | grep -q "^${CONTAINER_NAME}$"; then
+    echo "Stopping and removing existing container: $CONTAINER_NAME"
+    docker stop "$CONTAINER_NAME" || true
+    docker rm "$CONTAINER_NAME" || true
+else
```

Comment:
Please remove the else statement. I believe in this case we can follow the https://en.wikipedia.org/wiki/Talk%3AUnix_philosophy#No_news_is_good_news approach.